griptape/tasks/base_task.py

```diff
@@ -191,7 +191,7 @@ def run(self) -> BaseArtifact: ...
 
     @property
     def full_context(self) -> dict[str, Any]:
-        if self.structure:
+        if self.structure is not None:
```

Comment:
i know this is a small fix for this property, but why wouldnt this just return `self.context` if `self.structure` is None?

Reply:
That's in the [next PR](https://github.com/griptape-ai/griptape/pull/1224/files)